clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -107,6 +107,9 @@ def ZeroAttr : CIR_Attr<"Zero", "zero", [TypedAttrInterface]> {
 
   let parameters = (ins AttributeSelfTypeParameter<"">:$type);
   let assemblyFormat = [{}];
+  let extraClassDeclaration = [{
+    static llvm::StringRef getName() { return "cir.zero"; };
+  }];
```

Comment:
Are we gonna need to add an extra class declaration every time an attribute is translated to LLVM? Is it worth adding this in CIR_Attr directly and automatically build the string using `!strconcat(`?

Reply:
@bcardosolopes did a small refactor to not need this method at all. Instead, it now uses type-checking.